internal/transport/transport_test.go

```diff
@@ -2618,6 +2618,10 @@ func (s) TestClientDecodeHeaderStatusErr(t *testing.T) {
 		metaHeaderFrame *http2.MetaHeadersFrame
 		// output
 		wantStatus *status.Status
+		// end stream output
```

Comment:
I'm not sure what this means.

Let's make sure all our comments are useful and easy to understand, or just remove the comment if it's mostly just redundant.